packages/store/ts/config/v2/output.ts

```diff
@@ -59,7 +59,8 @@ export type Codegen = {
    */
   // TODO: move `namespaces` key handling into store so we can conditionally turn this on/off
   readonly namespaceDirectories: boolean;
-  readonly indexFilename: string;
+  // @deprecated import table libraries from their individual files instead */
```

Comment:
should check that this strikesthrough `indexFilename` usage, I can't recall if tsdoc tags needed to be in a `/* */` style comment or can be in any comment

(if we leave the `//` style comment, should remove the `*/` at the end)

Reply:
Fixed now, it does need to be `/* */`!